pep-9990.rst

```diff
@@ -0,0 +1,129 @@
+PEP: ?
+Title: Specification of default extras in package metadata
+Author: Ronny Pfannschmidt <opensource@ronnypfannschmidt.de
+Sponsor:
```

Comment:
I'm happy to sponsor this as a PEP editor, though I'm not a packaging expert. The sponsor role is primarily about helping you through the PEP process, as noted in https://peps.python.org/pep-0001/#submitting-a-pep

```suggestion
Sponsor: Adam Turner <python@quite.org.uk>
```